connectors/ssh/src/main/java/org/crsh/ssh/term/inline/SSHInlineCommand.java

```diff
@@ -58,8 +59,17 @@ public void destroy() {
   public void run() {
 
     // get principal
-    PrintStream err = new PrintStream(this.err);
-    PrintStream out = new PrintStream(this.out);
+    PrintStream err = null;
+    PrintStream out = null;
+    try {
+      err = new PrintStream(this.err, false, "UTF-8");
+      out = new PrintStream(this.out, false, "UTF-8");
```

Comment:
It would be better if this was settable, rather than forcing an encoding, especially if the default charset is not desired.